JASP-Desktop/widgets/listmodeljagsdatainput.cpp

```diff
@@ -164,10 +171,10 @@ void ListModelJAGSDataInput::initValues(OptionsTable * bindHere)
 	if (_values.size() > 0 && int(_values[0].size()) != _rowNames.size())
 		Log::log() << "Number of rows specifed in Options for ListModelJAGSDataInput does not match number of rows in values!" << std::endl;
 
-
 	beginResetModel();
 
-	_columnCount = _colNames.size();
+	_rowCount		= _rowNames.size();
```

Comment:
This line solves it, because `_rowCount` wasn't the right size `_values` wasn't the right size I think, which caused it to crash.

I think we should clean up these model a bit more by the way, and make sure that the size is only set in 1 location. (Not in _rowNames.length and _rowCount for instance)